nucypher/cli/literature.py

```diff
@@ -452,105 +452,102 @@
 # Worklock
```

Comment:
:+1: I am glad we isolated this file now :-)

Reply:
I totally agree. I remember I felt some resistance at the moment but now I'm also glad we made the change.